### PR TITLE
doc(deploy): mount host's directory /boot

### PR DESCRIPTION
**Pull request by mantissahz**
#### Which issue(s) this PR fixes:

Issue # longhorn/longhorn#9153

#### What this PR does / why we need it:

Update the document that we add a new directory.

### CODE REVIEW

**Comment by derekbit** [comment]
@jillian-maroket @jhkrug Please help review the PR. Thanks.

File: content/docs/1.7.2/deploy/install/_index.md [hunk 1]
```diff
@@ -156,6 +156,7 @@ Longhorn components require root access with privileged permissions to achieve v
 Below are the directories Longhorn components requiring access with root and privileged permissions :
 
 - Longhorn Manager
+  - /boot: Get required modules' information from /boot/config-$(uname -r) on the host.
```
Comment (innobead) [inline content/docs/1.7.2/deploy/install/_index.md:159]: Please make sure the mount is read-only in the implementation.

It seems more host mounts are required for environment check purposes. We need to ensure the least privilege.

Comment (innobead) [reply content/docs/1.7.2/deploy/install/_index.md:159]: It seems we are adding similar capacities of CLI preflight to the longhorn-manager. Will this cause duplicated implementation? Why not just use CLI directly?

Comment (derekbit) [reply content/docs/1.7.2/deploy/install/_index.md:159]: CLI preflight check is usually executed before installing Longhorn system, and longhorn-manager node.status.condition is for runtime self-diagnosis. Although they check the same kernel modules and utilities, they serve different purposes.

Comment (derekbit) [reply content/docs/1.7.2/deploy/install/_index.md:159]: WDYT? @innobead @c3y1huang

Comment (innobead) [reply content/docs/1.7.2/deploy/install/_index.md:159]: This has been discussed with @derekbit . What I care about is only duplicated implementation, so if we can achieve that in the same shared code eventually, that will be good.

**Review by jillian-maroket — APPROVED** [review]

File: content/docs/1.7.2/deploy/install/_index.md (hunk 1 above)
Comment (c3y1huang) [reply content/docs/1.7.2/deploy/install/_index.md:159]: https://github.com/longhorn/longhorn/issues/9933

**Review by c3y1huang — APPROVED** [review]

**Pull request merged**